src/net/wooga/jenkins/pipeline/TestHelper.groovy

```diff
@@ -68,6 +68,11 @@ def transformIntoCheckStep(platform, testEnvironment, coverallsToken, config, ch
           }
         }
       }
+      catch (Exception e){
+        if (errorHandler) {
+          errorHandler.call(e)
+        }
+      }
```

Comment:
If you don't have an errorHandler then we need to throw the exection otherwise every test would become non failing.

Reply:
Ah yes, my bad.